### ISSUE TITLE
Missing information in PoS graph when analyzing "C'est"

### ISSUE THREAD

**Issue by simonmarchal** [aymara/lima#52]
After analyzing "C'est un test.", the output of the analysis deletes "C'est" :
3       un      un      DET     DET     _       _       4       DETSUB  _       _
4       test    test    NC      NC      _       _       _       _       _       _
5       .       .       PONCTU  PONCTU_FORTE    _       _       _       _       _       _

Before the syntactic analysis, the analysis graph and the PoS graph are correct (see attached files test-1.txt.bp.dot.png, test-1.txt.dot.png).
After the syntactic analysis, the morpho syntactic data of "C'" and "est" nodes are corrupted (see attached file test-1.txt.afterSA.dot.png).
![test-1 txt aftersa dot](https://cloud.githubusercontent.com/assets/17700014/19066757/2b88b96e-8a1b-11e6-8000-3b47a161ab6d.png)
![test-1 txt bp dot](https://cloud.githubusercontent.com/assets/17700014/19066759/2b8bacd2-8a1b-11e6-9651-ab7291a0037c.png)

![test-1 txt dot](https://cloud.githubusercontent.com/assets/17700014/19066758/2b8977fa-8a1b-11e6-87fe-2c920fee2b41.png)

**Comment by kleag** [comment]
The problem comes from a parsing rule constraint:

```+EnforcePropertiesConstraints(trigger.1,right.3,"NUMBER,PERSON")```

inside the rule:

```
@PronPersSuj::(@NegPrem)? (@PronPersCodPrev|@PronPersCoiPrev|@PronRefl){0-2} @Verbe:SYNTACTIC_RELATION:
+SecondUngovernedBy(trigger.1,right.3,"ANY")
+NumberAgreement(trigger.1,right.3)
+EnforcePropertiesConstraints(trigger.1,right.3,"NUMBER,PERSON")
+CreateRelationBetween(trigger.1,right.3,"PronSujVerbe")
=>AddRelationInGraph()
=<ClearStoredRelations()

```

in rules-fre-homodeps-pass1.txt

I will now check the code.

**Comment by kleag** [comment]
Corrected in commit 106c605 .
